docs/resources/custom_certificate.md

```diff
@@ -0,0 +1,44 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "vercel_custom_certificate Resource - terraform-provider-vercel"
+subcategory: ""
+description: |-
+  Provides an Custom Certificate Resource, allowing Custom Certificates to be uploaded to Vercel.
+  By default, Vercel provides all domains with a custom SSL certificates. However, Enterprise teams can upload their own custom SSL certificate.
+  For more detailed information, please see the Vercel documentation https://vercel.com/docs/domains/custom-SSL-certificate.
+---
+
+# vercel_custom_certificate (Resource)
+
+Provides an Custom Certificate Resource, allowing Custom Certificates to be uploaded to Vercel.
+
+By default, Vercel provides all domains with a custom SSL certificates. However, Enterprise teams can upload their own custom SSL certificate.
+
+For more detailed information, please see the [Vercel documentation](https://vercel.com/docs/domains/custom-SSL-certificate).
+
+## Example Usage
+
+```terraform
+resource "vercel_custom_certificate" "example" {
+  private_key                       = file("private.key")
+  certificate                       = file("certificate.crt")
+  certificate_authority_certificate = file("ca.crt")
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `certificate` (String) The certificate itself. Should start with -----BEGIN CERTIFICATE----- and end with -----END CERTIFICATE-----
```

Comment:
can specify "in PEM format" which covers the BEGIN/END CERTIFICATE lines and base64